launch/Mako_G-319.launch

```diff
@@ -1,6 +1,6 @@
 <launch>
   <arg name="name" default="camera" doc="The name of the camera"/>
-  <arg name="frame_id" default="camera" doc="The frame id of the camera"/>
+  <arg name="frame_id" default="$(arg name)" doc="The frame id of the camera"/>
   <arg name="ip" default="" doc="The IP for the camera to connect to"/>
   <arg name="guid" default="" doc="The GUID for the camera to connect to"/>
   <arg name="camera_info_url" default="file://$(find avt_vimba_camera)/calibrations/calibration_50-0503343289.yaml"/>
```

Comment:
What is the meaning of "50-0503343289" here and the find command in stereo_camera.launch?

Reply:
So this is the calibration file for some camera with a GUID of `50-0503343289` that the original author of the repo had.
The calibration file contains the intrinsic calibration which gets published along with the image for rectification or un-distortion purposes.
Good point though, I'm going to rename the file to "example" or something to reduce confusion.